explorer/src/components/account/MetadataAttributesCard.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react'
+import { NFTData } from 'providers/accounts'
+import { LoadingCard } from 'components/common/LoadingCard'
+import { ErrorCard } from 'components/common/ErrorCard'
+
+interface Attribute {
+  trait_type: string
+  value: string
+}
+
+export function MetadataAttributesCard({ nftData }: { nftData: NFTData }) {
+  const [attributes, setAttributes] = React.useState<Attribute[]>([])
+  const [status, setStatus] = React.useState<'loading' | 'success' | 'error'>(
+    'loading'
+  )
+
+  async function fetchMetadataAttributes() {
+    try {
+      const response = await fetch(nftData.metadata.data.uri)
+      const metadata = await response.json()
+      if (Array.isArray(metadata.attributes)) {
+        setAttributes(metadata.attributes)
```

Comment:
Attributes could be anything, can you filter for whether they have the expected attributes `trait_type` and `value`?